functable.c

```diff
@@ -129,6 +129,7 @@ static void init_functable(void) {
     if (cf.x86.has_avx512_common) {
         ft.adler32 = &adler32_avx512;
         ft.adler32_fold_copy = &adler32_fold_copy_avx512;
+        ft.inflate_fast = &inflate_fast_avx512;
```

Comment:
Missing assignment for AVX-512 dynamic dispatching:

```c
ft.chunkmemset_safe = &chunkmemset_safe_avx512;
ft.chunksize = &chunksize_avx512;
```

Reply:
Good catch.  Fixed.